spec/lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider_spec.rb

```diff
@@ -14,7 +14,7 @@
   # BDD Document Type
   let(:va_document_type) { 'L023' }
 
-  let(:provider) do
+  let!(:provider) do
```

Comment:
TLDR these need to initialized for these tests to avoid some weird situations with metrics logging in callback chains when setting up submissions to test with (e.g. logging when certain objects are created). Without getting into the weeds here, doing this in both provider tests allows us to more reliably test the metrics logging at the point we want to test it here.